e2e/Procfile

```diff
@@ -0,0 +1,3 @@
+grap: trap 'podman stop graphite-exporter; podman rm graphite-exporter;' EXIT > /dev/null; podman run --name graphite-exporter --network host --rm prom/graphite-exporter --web.listen-address="127.0.0.1:9180"
+prom: trap 'podman stop prometheus; podman rm prometheus;' EXIT > /dev/null; podman run --name prometheus --network host --rm -v $PWD/theus:/etc/prometheus prom/prometheus --config.file=/etc/prometheus/config.yaml --web.listen-address="127.0.0.1:9090"
```

Comment:
Are `graphite-exporter` and `prometheus` expected to be installed when running this file? Can't find references to them in the README, do they need to be enumerated there?

Reply:
That might be the issue I'm running into :thinking:

Reply:
It's clown-native